AndroidAsync/src/com/koushikdutta/async/http/body/FileBody.java

```diff
@@ -27,10 +34,13 @@ public void parse(DataEmitter emitter, CompletedCallback completed) {
         throw new AssertionError("not implemented");
     }
 
-    public static final String CONTENT_TYPE = "application/binary";
```

Comment:
I'd prefer you leave the default content type here. Will submit another change to address this.